### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by andriy-shymkiv**

### CODE REVIEW

**Review by spiritkyiv — APPROVED** [review]
looks good

File: src/createServer.js [hunk 1]
```diff
@@ -1,3 +1,68 @@
-// Write code here
-// Also, you can create additional files in the src folder
-// and import (require) them here
+/* eslint-disable max-len */
```
Comment (mykola-koval) [inline src/createServer.js:1]: you can use string concatenation to omit this

File: src/createServer.js [hunk 2]
```diff
@@ -1,3 +1,68 @@
...
+
+const http = require('http');
+const { convertToCase } = require('./convertToCase');
+
+const createServer = () => {
+  const server = http.createServer((req, res) => {
+    const [pathName, queryParams] = req.url.split('?');
+    const text = pathName.slice(1);
+    const params = new URLSearchParams(queryParams);
```
Comment (mykola-koval) [inline src/createServer.js:10]: you can use `new URL` constructor to simplify this logic

File: src/createServer.js [hunk 3]
```diff
@@ -1,3 +1,68 @@
...
+      });
+
+      res.setHeader('Content-Type', 'application/json');
+      res.statusCode = 400;
+      res.statusText = 'Bad request';
+
+      res.end(errorResponse);
+
+      return;
+    }
```
Comment (mykola-koval) [inline src/createServer.js:46]: let's move error handling to a separate file

File: src/createServer.js [hunk 4]
```diff
@@ -1,3 +1,68 @@
...
+    res.statusCode = 200;
+
+    const formattedResponse = JSON.stringify({
+      originalCase: result.originalCase,
+      targetCase: toCase,
+      originalText: text,
+      convertedText: result.convertedText,
+    });
+
+    res.end(formattedResponse);
```
Comment (mykola-koval) [inline src/createServer.js:60]: as well as sending response

**Review by mykola-koval — APPROVED** [review]

**Pull request closed without merging**